packages/core/admin/ee/admin/permissions/customPermissions.js

```diff
@@ -1,5 +1,9 @@
 const customPermissions = {
   settings: {
+    reviewWorkflows: {
+      main: [{ action: 'admin::roles.create', subject: null }],
```

Comment:
Now that you've figured this out, is it worth adding a _very very very_ small bit of documentation at the same time about how to add a route (or more specifically EE route)? 🙏🏼 pls 🙏🏼

Reply:
I thought about it too yesterday, but I've got two problems:

- this would be more of an example/ guide and I'm not sure if we really want to maintain guides like that in the contrib docs. 
- I'd find it more valuable to document how RBAC works in the FE, but I'm lacking a lot of knowledge there

However: I like the idea. What about we set up a meeting among FE and collaboratively write some docs on this topic?